spatial-features-h3/index.ts

```diff
@@ -0,0 +1,105 @@
+import './style.css';
+import 'maplibre-gl/dist/maplibre-gl.css';
+import maplibregl from 'maplibre-gl';
+import {Deck} from '@deck.gl/core';
+import {H3TileLayer, h3QuerySource, BASEMAP, colorBins} from '@deck.gl/carto';
+
+const cartoConfig = {
+  apiBaseUrl: import.meta.env.VITE_API_BASE_URL,
+  accessToken: import.meta.env.VITE_API_ACCESS_TOKEN,
+  connectionName: 'carto_dw'
+};
+
+const INITIAL_VIEW_STATE = {
+  latitude: 40.7128, // Aproximado para Nueva York
+  longitude: -74.006, // Aproximado para Nueva York
+  zoom: 7,
+  pitch: 0,
+  bearing: 0,
+  minZoom: 3.5,
+  maxZoom: 15
+};
+
+// Selectors
+let selectedVariable = 'population';
+
+const variableSelector = document.getElementById('variable') as HTMLSelectElement;
+variableSelector?.addEventListener('change', () => {
+  selectedVariable = variableSelector.value;
+  render();
+});
+
+function initSelectors() {
+  const variableSelector = document.getElementById('variable') as HTMLSelectElement;
+  fetch(
+    'https://public.carto.com/api/v4/data/observatory/metadata/datasets/cdb_spatial_fea_94e6b1f/variables?minimal=true'
+  )
+    .then(response => response.json())
+    .then(data => {
+      const options = data
+        .filter(variable => variable.db_type === 'FLOAT')
+        .map((variable: any) => {
```

Comment:
Can we type this? At least with the minimal columnName and variableName.

Reply:
Done. Typed in utils